package.json

```diff
@@ -1,38 +1,53 @@
 {
-  "name": "ResistanceCalendar",
-  "version": "1.0.0",
-  "repository": "https://github.com/ResistanceCalendar/resistance-calendar-frontend.git",
-  "main": "server.js",
+  "name": "resistance-calendar-frontend",
+  "version": "0.1.0",
   "license": "MIT",
+  "scripts": {
+    "build": "rimraf dist && webpack -p",
+    "dev": "webpack-dev-server",
+    "test": "jest --config=testConfig/jest.config.json --watch",
```

Comment:
Good point.  To be consistent with the linting scripts, would `test` and `test:watch` work?